src/app/ipv/middleware.test.js

```diff
@@ -562,17 +562,17 @@ describe("journey middleware", () => {
   });
 
   context(
-    "handleMultipleDocCheck: handling journey action with ukPassport, drivingLicence, end",
+    "handleJourneyAction: handling journey action with ukPassport, drivingLicence, end",
     () => {
       it("should postJourneyEvent with ukPassport", async function () {
         req = {
           id: "1",
-          body: { journey: "next/passport" },
+          body: { journey: "ukPassport" },
```

Comment:
Do we need to sync this change with core-back?

Reply:
Ah, no since the ukPassport is the event core-back wants - it's just the `handleMultipleDocCheck` would take `next/passport` and call core-back with `ukPassport`